testroot

```diff
@@ -0,0 +1,3 @@
+allow_warnings = false
```

Comment:
what testing will use this?
the allow_warnings = false is more restrictive than all the apps are

Reply:
The opt testing. We can remove it, I don't mind